pyproject.toml

```diff
@@ -120,6 +120,8 @@ lint = [
 mypy = [
```

Comment:
This stuff is marked in red here, do you also have this? Does this mean anything?
![image](https://github.com/emdgroup/baybe/assets/84016733/93c38f3d-fc9d-4e56-be35-a0cf05e89f09)

Reply:
Hm, good question! No idea, honestly! Did not encounter any issues. @Scienfitz?

Reply:
validated again with https://www.toml-lint.com/
seems `just github things`
ignore